section11/numberformat_proposed.html

```diff
@@ -273,6 +228,74 @@ <h1>PartitionNumberPattern ( _numberFormat_, _x_ )</h1>
             1. Append a new Record { [[Type]]: *"unknown"*, [[Value]]: _unknown_ } as the last element of _result_.
         1. Return _result_.
       </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-getnotationsubpattern" aoid="PartitionNotationSubPattern">
+      <h1>PartitionNotationSubPattern ( _numberFormat_, _x_, _n_, _exponent_ )</h1>
+      <p>
+        The PartitionNotationSubPattern abstract operation is called with arguments _numberFormat_ (which must be an object initialized as a NumberFormat), _x_ (which is a numeric value after rounding is applied), _n_ (which is an intermediate formatted string), and _exponent_ (an integer), and creates the corresponding parts for the number and notation according to the effective locale and the formatting options of _numberFormat_. The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _result_ be a new empty List.
+        1. If _x_ is *NaN*, then
+          1. Append a new Record { [[Type]]: *"nan"*, [[Value]]: _n_ } as the last element of _result_.
+        1. Else if _x_ is not a finite Number or BigInt,
```

Comment:
for spec steps this looks a bit informal, I wonder if we should have a Type(_x_) check for BigInt. Although, this might just expand the text.

Reply:
This is pre-existing text cut and pasted from above.